app/pages/clinicworkspace/TideDashboardV2/filters/AppliedFiltersList.js

```
@@ -0,0 +1,78 @@
+  const activeFilters = { lastDataType: 'cgm', lastData, clinicSites, patientTags };
+
+  const handleResetFilters = () => {
+    dispatch(setPatientTagsFilter([]));
+    dispatch(setClinicSitesFilter([]));
+    dispatch(setOffset(0));
+  };
+
+  const handleRemoveFilter = (filterKey, value) => {
+    switch (filterKey) {
+      case 'patientTags':
+        const updatedTags = without(patientTags, value);
```

Biome (2.5.8)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)